pages/content_packs.rst

```diff
@@ -0,0 +1,101 @@
+*************
+Content Packs
+*************
+
+What are content packs?
+=======================
+
+Content packs are a convenient way to share configuration. A content pack is a
+JSON file which contains a set of configurations of Graylog components.
+This JSON file can be uploaded to Graylog instances and then installed.
+A user who took the time to create a input, pipelines and dashboard for a
+certain type of log format, can so easily share his efforts with the community.
+
+Content packs can be found on the `Graylog Marketplace <https://marketplace.graylog.org/>`__.
+
+Parameter
+---------
+
+Content packs can have parameters. Those parameters help to adjust the
+configuration to the needs of the user. A good example usage is the port of
+an input. The creator of the content pack may have his input running on port
+55055, but the user of the content pack may have already an input running on
+that specific port.
+The creator can specify a parameter and assign it to the port.
+The user of the content pack will be asked for a value of the parameter on
+installation. The value than will be used as the port of the input.
```

Comment:
Last sentence is a bit unclear, try instead, "The provided value will then be used as the port of the input on the new system."